Makefile

```diff
@@ -7,26 +7,23 @@ RSRC=$(HOME)/go/bin/rsrc
 TESTS=naksu/mebroutines naksu/mebroutines/backup naksu naksu/box naksu/boxversion naksu/network
 
 bin/golangci-lint:
-	# 1.15.0 is latest officially supporting go 1.10
-	# however, it fails to run with -GOOS-windows (https://github.com/golangci/golangci-lint/issues/323)
-	# thus, we use 1.16.0
-	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./bin v1.16.0
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ./bin v1.30.0
 
 bin/go2xunit:
 	GOPATH=$(current_dir)/ go get github.com/tebeka/go2xunit
 
 checkstyle: bin/golangci-lint
-	-GOOS=linux GOARCH=amd64 CGO_ENABLED=1 ./bin/golangci-lint run --out-format checkstyle ./src/naksu/... > checkstyle-linux.xml
-	-GOOS=windows GOARCH=amd64 CGO_ENABLED=1 ./bin/golangci-lint run --out-format checkstyle ./src/naksu/... > checkstyle-windows.xml
+	-cd src/naksu && GOOS=linux GOARCH=amd64 CGO_ENABLED=1 GOPATH=$(current_dir)/ ../../bin/golangci-lint run --timeout 5m0s --out-format checkstyle > $(current_dir)/checkstyle-linux.xml
+	-cd src/naksu && GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc CXX=x86_64-w64-mingw32-g++ GOPATH=$(current_dir)/ ../../bin/golangci-lint run --timeout 5m0s --out-format checkstyle > $(current_dir)/checkstyle-windows.xml
 
 lint: bin/golangci-lint
-	GOPATH=$(current_dir) ./bin/golangci-lint run --out-format checkstyle ./src/naksu/...
+	cd src/naksu && GOPATH=$(current_dir) ../../bin/golangci-lint run --out-format checkstyle
 
 ci-test: bin/go2xunit
-	2>&1 GOPATH=$(current_dir)/ go test -v $(TESTS) | ./bin/go2xunit -output tests.xml
+	cd src/naksu && 2>&1 GOPATH=$(current_dir)/ go test -v $(TESTS) | ../../bin/go2xunit -output $(current_dir)/tests.xml
 
 test:
-	GOPATH=$(current_dir)/ go test $(TESTS)
+	cd src/naksu && GOPATH=$(current_dir)/ go test $(TESTS)
 
 docker: clean
```

Comment:
`make docker` voisi vielä muiden tiedostojen ohella kopioida `src/naksu/go.sum`- ja `src/naksu/go.mod`-tiedostot paikoilleen, jotta nämä päivittyvät myös tätä komentoa ajaessa (jos uusia riippuvuuksia on lisätty). Nyt esim. `make naksu` päivittää nämä, mutta `make docker` ei.

Reply:
Ymmärsinkö oikein, että buildin jälkeen niinkuin `Copy go modules list back from docker`

Reply:
Just näin. Lähinnä ettei kävisi niin helposti sitä samaa kuin depin kanssa, että moduulilistat pitää muistaa erikseen päivittää käsin.